hugo/layouts/papers/single.html

```diff
@@ -496,14 +508,14 @@ <h5 class="card-title">Abstract</h5>
       {{ end }}
       <dt class="acl-button-row">Copy Citation:</dt>
       <dd class="acl-button-row">
-      {{ if $has_bib }}
+      {{ with $paper.bibtex }}
         <button type="button" class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target="#citeBibtexContent"><i class="far fa-clipboard pr-2"></i>BibTeX</button>
       {{ end }}
```

Comment:
This button doesn’t appear for me on the preview, but it looks like this is because clipboard-js doesn’t work on the preview. That appears to be a problem for _all_ preview branches, should probably fix this...

Reply:
Fixed